common/build/build-common/tsconfig.esm.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": ["./tsconfig.base.json"],
+	"compilerOptions": {
+		"module": "ESNext",
```

Comment:
Mixed space and tab indent. If our formatter doesn't give an error for this, we should fix that.

Reply:
Good catch. This package is unique because it has no build, where the formatting would normally be caught. I updated the pipeline to run prettier as a "check" which should address this.